overcooked_ai_py/agents/agent_test.py

```diff
@@ -66,316 +66,311 @@ def test_fixed_plan_agents(self):
-        start_state.objects = {(4, 2): Obj('soup', (4, 2), ('onion', 2, 0)),
-                               (4, 3): Obj('soup', (4, 3), ('onion', 3, 5))}
-        eva.start_state = start_state
-        self.compare_times(eva, h_idx=0)
+# Deprecated. Due to CoupledPlanningAgent
+# class TestExtraAgents(unittest.TestCase):
+#
+#     def setUp(self):
+#         self.mlp_large = MediumLevelPlanner.from_pickle_or_compute(large_mdp, NO_COUNTERS_PARAMS, force_compute=force_compute_large)
+#
+#     def test_two_coupled_agents_coupled_pair(self):
+#         mlp_simple = MediumLevelPlanner.from_pickle_or_compute(simple_mdp, NO_COUNTERS_PARAMS, force_compute=force_compute)
+#         cp_agent = CoupledPlanningAgent(mlp_simple)
+#         agent_pair = CoupledPlanningPair(cp_agent)
+#         start_state = OvercookedState(
+#             [P((2, 2), n),
+#              P((2, 1), n)],
+#             {})
+#         env = OvercookedEnv.from_mdp(simple_mdp, start_state_fn=lambda: start_state)
+#         trajectory, time_taken, _, _ = env.run_agents(agent_pair, include_final_state=True, display=DISPLAY)
+#         end_state = trajectory[-1][0]
+#
+#     def test_one_coupled_one_fixed(self):
+#         a0 = CoupledPlanningAgent(self.mlp_large)
+#         a1 = FixedPlanAgent([s, e, n, w])
+#         agent_pair = AgentPair(a0, a1)
+#         env = OvercookedEnv.from_mdp(large_mdp, horizon=10)
+#         trajectory, time_taken, _, _ = env.run_agents(agent_pair, include_final_state=True, display=DISPLAY)
+#         self.assertEqual(time_taken, 10)
+#
+#     def test_one_coupled_one_greedy_human(self):
+#         # Even though in the first ~10 timesteps it seems like agent 1 is wasting time
+#         # it turns out that this is actually not suboptimal as the true bottleneck is
+#         # going to be agent 0 later on (when it goes to get the 3rd onion)
+#         a0 = GreedyHumanModel(self.mlp_large)
+#         a1 = CoupledPlanningAgent(self.mlp_large)
+#         agent_pair = AgentPair(a0, a1)
+#         start_state = OvercookedState(
+#             [P((2, 1), s),
+#              P((1, 1), s)],
+#             {})
+#         env = OvercookedEnv.from_mdp(large_mdp, start_state_fn=lambda: start_state)
+#         trajectory, time_taken, _, _ = env.run_agents(agent_pair, include_final_state=True, display=DISPLAY)
+#         end_state = trajectory[-1][0]
+#
+#     def test_embedded_planning_agent(self):
+#         agent_evaluator = AgentEvaluator({"layout_name": "cramped_room_o_1order"}, {"horizon": 100})
+#         other_agent = GreedyHumanModel(agent_evaluator.mlp)
+#         epa = EmbeddedPlanningAgent(other_agent, agent_evaluator.mlp, agent_evaluator.env, delivery_horizon=1)
+#         ap = AgentPair(epa, other_agent)
+#         agent_evaluator.evaluate_agent_pair(ap, num_games=1, display=DISPLAY)
+
+
+# Deprecated. Due to CoupledPlanningAgent
+# class TestScenarios(unittest.TestCase):
+#
+#     """
+#     Corridor: assuming optimality / planning horizon – scenario1_s
```

Comment:
I feel like these Scenario tests can be totally deleted, there is close to 0 chance that we will be revisiting this

Reply:
Removing these in the newest commit.